src/System.Linq.Parallel/tests/Helpers/CustomPartitioners.cs

```diff
@@ -0,0 +1,164 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading;
+
+namespace System.Linq.Parallel.Tests
+{
+    // This class is meant to be used with partitioners that do load balancing, but _not_ buffering.
+    // Feeding it a buffering partitioner will result in incorrect results!
+    internal sealed class StrictPartitioner<T> : Partitioner<T>
```

Comment:
FYI, this type didn't change at all, other than my marking it as sealed.  I think it's showing this way because I renamed the file while adding the below type to it.